games-action/polymc/polymc-9999.ebuild

```diff
@@ -68,6 +68,7 @@ DEPEND="
 # And we need more than just the GL headers
 RDEPEND="
 	${QT_DEPS}
+	>=dev-libs/quazip-1.3:=
 	sys-libs/zlib
```

Comment:
Maybe we can rename `QT_DEPS` to `COMMON_DEPEND` so we can add quazip/zlib etc in there as well and save some duplicate lines.

Reply:
I think having a separate `QT_DEPS` is nice for organization, but a `COMMON_DEPEND` that includes `QT_DEPS` seems nice!